Sebastian-Sole/handicappin
# ✨ feat: Homepage

## What was done
- Homepage added for logged in users with hero displaying stats, section for graphs, calculators and a little "about" section. 
- Navbar implemented with correct links
- Adjusted tailwind screen sizes to better facilitate mobile views
- Added footer
- Added card calculators for Course Handicap and Score Differential
- Added additional charts for handicap trend and score
- Fixed UI responsivity bug with login and sign up pages
- Changed Logout button UI
- Added shadcn components such as Sheet
- Added api query for "best round"

## How to test
- Login and see the homepage displayed with no data
- Add at least 5 rounds
- Revisit the homepage to see homepage displayed with user data
- Resize screen to see hamburger menu for navbar and sheet with links
- Test out the calculators at the bottom of the homepage

## Future work
- Add endpoints for navbar links and statistics
- Additional icons and stats in hero - what data should be here?
- "Performance breakdown" box - what should be here?
